tests/Tests/100__deploy/130__operators/130__rhods_operator/132__traefik-proxy_restart_verification.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation       132 - RHODS_TRAEFIK_PROXY_CONTAINER_RESTART_VERIFICATION
+...                 Verify that rhods traefik-proxy container
+...                 restart verification
+...
+...                 = Variables =
+...                 | Namespace         | Required |    RHODS Namespace/Project for RHODS operator POD |
+...                 | LABEL_SELECTOR    | Required |    Label selector for traefik proxy|
+
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Variables ***
+${NAMESPACE}            redhat-ods-applications
+${LABEL_SELECTOR}       name=traefik-proxy
+
+
+*** Test Cases ***
+Verify Traefik Proxy Containers Restart
+    [Documentation]    Verify traefik proxy
+    ...    container restart
+    [Tags]    Sanity
+    ...       ODS-1163    KnownIssues
+    ${p_names}    Get POD Names    ${NAMESPACE}    ${LABEL_SELECTOR}
+    Verify Restart Container Verification    ${p_names}
+
+
+*** Keywords ***
+Verify Restart Container Verification
```

Comment:
I think, it would be better to have this keyword more generic. Let's say, `Verify Containers Restarts Are Less Than A Threshold` and it would get "pod name" and "threshold" as arguments.

As alternative, `Verify Containers Have Zero Restarts`

Reply:
ohh nice..that makes more sense thanks !! will change the name. We can keep threshold for restart